apps/test/netsim/NetSimVizNode.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+/* global describe */
+/* global beforeEach */
+/* global it */
+/* global $ */
+
+var testUtils = require('../util/testUtils');
+testUtils.setupLocale('netsim');
+var assert = testUtils.assert;
+var assertEqual = testUtils.assertEqual;
+var netsimTestUtils = require('../util/netsimTestUtils');
+var fakeShard = netsimTestUtils.fakeShard;
+var assertTableSize = netsimTestUtils.assertTableSize;
+
+var NetSimVizNode = require('@cdo/apps/netsim/NetSimVizNode');
+
+describe("NetSimVizNode", function () {
```

Comment:
Agree! Also fine to put off until next work on Netsim if it'd be time consuming / unlikely to regress in the interim